packages/amplify-cli-core/src/hooks/hooksExecutor.ts

```diff
@@ -74,7 +74,15 @@ const execHelper = async (
 
   try {
     logger.info(`hooks file: ${execFileMeta.fileName} execution started`);
-    const childProcess = execa(runtime, [execFileMeta.filePath], {
+    let runtimeArgs;
+    // addings runtime options passed in hooks_config.json
+    if(Array.isArray(hooksRuntime.runtime_options) && hooksRuntime.runtime_options.length > 0){
```

Comment:
why are we checking isArray if we typed it as array in type definition?
wouldn't `if(hooksRuntime.runtime_options)` suffice? concatenating empty array should be fine.